packages/teams-js/test/public/dialog.spec.ts

```diff
@@ -131,9 +132,9 @@ describe('Dialog', () => {
 
       let callbackCalled = false;
       const dialogInfo: DialogInfo = {};
-      dialog.open(dialogInfo, (err, result) => {
-        expect(err).toBe('someError');
-        expect(result).toBeUndefined();
+      dialog.open(dialogInfo, resultObj => {
```

Comment:
I don't know much about our end to end testing: do we also need to update those tests as part of this change? Can we do it in the same checkin as this?

Reply:
Yes, and it is on my list. I did add the new open function to test-app. I will add the tests on the other end as well but before that, I need to rename existing tests (renaming will fail the pipelines, so I have to figure some way).